src/__tests__/Navigation.test.js

```diff
@@ -0,0 +1,10 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navigation from '../components/Navigation';
+
+it('renders correctly', () => {
+  const navigation = renderer
+    .create(<Router><Navigation /></Router>)
```

Comment:
- [ ] Great job on the home page so far but I think it would be great if you stich with the design guidelines and made the layout as required so please add the section between the crypto divs and the nav bar as I will attach an image here to understand. you made it in the details page great jsut add the section in the home page please! 🚀💯✨✅

![image](https://user-images.githubusercontent.com/58553711/210929111-9203a877-7780-430c-a05d-09a4f18730b8.png)


- Your design
![image](https://user-images.githubusercontent.com/58553711/210929550-986f67d8-5536-4eec-8b89-b10e01c9d818.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/210929594-5eb54e9c-25cd-442f-bda4-b9ad69c237a7.png)